pkg/pillar/cmd/baseosmgr/baseosmgr.go

```diff
@@ -28,17 +26,12 @@ const (
 	warningTime = 40 * time.Second
```

Comment:
This is no longer used, right? Since it is the same as the default?
Need to make sure we don't end up with confusing duplication and dead code.